### DISCUSSION TITLE
Lesson 7 -m command for forge test

### DISCUSSION THREAD

**Discussion by DeanKrupka** [Cyfrin/foundry-full-course-cu#1085] [Q&A] [answered]
Error tells me -m is not a recognized command. This is used at minute-mark 00:27:34 of video 2. Patrick runs the command:

forge test -m testPriceFeedVersionIsAccurate -vvv --fork-url $SEPOLIA_RPC_URL

Just to be clear. I am asking why is -m working for Patrick (and not for me)?

**Comment by 0xusmanf** [comment, marked as answer]
`-m` command has been updated to `--mt`, that's why its not working for you. Use `--mt`. Also keep an eye on [this file](https://github.com/Cyfrin/foundry-full-course-f23/blob/main/chronological-updates.md), it has all the updates you should know through out the course.

**Reply by DeanKrupka** [reply]
Thank you!
